app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require './lib/chitter'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
```

Comment:
nice RESTful routes!  Did you consider combining `get /peeps` and `get /peeps/matches`?

Reply:
how could you do that?